content/docs/studio/get-started.md

```diff
@@ -1,13 +1,15 @@
 # Get Started with Iterative Studio
 
-The following video provides you a quick overview of how to use Iterative
+The following two videos provide you a quick overview of how to use Iterative
 Studio.
 
 > Note that we have renamed DVC Studio to Iterative Studio and Views to
 > Projects.
 
 https://www.youtube.com/watch?v=hKf4twg832g
 
+TODO: Add a video about live metrics tracking
```

Comment:
For now maybe make these HTML comments if you're concerned about accidental merges 🙂 

```suggestion
<!-- TODO: Add a video about live metrics tracking -->
```

Reply:
@jorgeorpinel I changed this everywhere, but then reverted it. When reviewing the docs by running it locally, it became harder to review coz it was not clear which images/videos were going to be replaced (coz the TODO text didn't display - as expected). An accidental merge will cause more issues than this anyway 🙂